caniusepython3/__main__.py

```diff
@@ -43,7 +43,10 @@ def projects_from_requirements(requirements):
             elif req.editable:
                 log.warning(
                     'Skipping {0}: editable projects unsupported'.format(req.name))
-            elif req.url and req.url.startswith('file:'):
+            elif getattr(req, 'url', None) and req.url.startswith('file:'):
+                log.warning(
+                    'Skipping {0}: file-specified projects unsupported'.format(req.name))
+            elif getattr(req, 'link', None) and req.link.scheme == 'file':
```

Comment:
Since the log warning is the same there is no need to make this two separate `elif` statements.